fontir/src/coords.rs

```diff
@@ -44,21 +45,27 @@ trait Converter<From, To> {
 ///
 /// Stores [PiecewiseLinearMap]'s in several directions. Sources
 /// suggest <= 10 mappings is typical, we can afford the bytes.
-#[derive(Debug, Clone)]
+#[derive(Serialize, Deserialize, Debug, Clone, PartialEq, Eq)]
+#[serde(from = "CoordConverterSerdeRepr", into = "CoordConverterSerdeRepr")]
 pub struct CoordConverter {
-    user_to_design: PiecewiseLinearMap,
+    pub(crate) default_idx: usize,
+    pub(crate) user_to_design: PiecewiseLinearMap,
     design_to_user: PiecewiseLinearMap,
     design_to_internal: PiecewiseLinearMap,
     internal_to_design: PiecewiseLinearMap,
 }
 
 impl CoordConverter {
+    pub fn nop() -> CoordConverter {
+        CoordConverter::from_user_to_design_examples(Vec::new(), 0)
```

Comment:
a no-op CoordConverter (to be used when a designspace axis has no explicit axis mappings) should return behave as if user-space and design-space are the same space, so like an identity mapping.
However internal normalized coordinates must always by definition be mapped to -1(min)..0(default)..+1(max).
So you always need the full user-space triplet of axis minimum, default and maximum (the fvar specifies those) to be able to normalize coordinates accordingly.
The way you made your default CoordConverter simply returns maps any number to itself, IIUC. That may be ok for user<=>design, but not for normalized.

Reply:
Hm. That might invalidate the entire concepts of a nop converter as there is no information to make that mapping.

Reply:
OK, you are right, it doesn't actually make sense. Deleted.